docs/how-to-guides/themes/block-theme-overview.md

```diff
@@ -124,6 +125,46 @@ As we're still early in the process, the number of blocks specifically dedicated
 
 One of the most important aspects of themes (if not the most important) is the styling. While initially you'll be able to provide styles and enqueue them using the same hooks themes have always used, the [Global Styles](/docs/how-to-guides/themes/theme-json.md) effort will provide a scaffolding for adding many theme styles in the future.
 
+## Internationalization (i18n)
+
+A pattern block can be used to insert translatable content inside a block template. Since those files are php-based, there is a mechanism to mark strings for translation or supply dynamic URLs.
+
+#### Example
+
+Register a pattern: 
+
+```php
+<?php
+register_block_pattern( 
+	'myblocktheme/wordpress-credit',
+	array(
+		'title'      => __( 'Wordpress credit', 'myblocktheme' ),
+		'content'    => '
+						<!-- wp:paragraph -->
+						<p>' .
+						sprintf(
+							/* Translators: WordPress link. */
+							esc_html__( 'Proudly Powered by %s', 'myblocktheme' ),
+							'<a href="' . esc_url( __( 'https://wordpress.org', 'myblocktheme' ) ) . '" rel="nofollow">WordPress</a>'
+						) . '</p>
+						<!-- /wp:paragraph -->',
+		'inserter'   => false
```

Comment:
Should we highlight this flag (`inserter=>false`) that hides such patterns from the list of displayed patterns in inserter/explorer. etc...?

Reply:
Yeah that would be super helpful